code/modules/unit_tests/unit_test.dm

```diff
@@ -124,6 +124,16 @@ GLOBAL_VAR_INIT(focused_test, focused_test())
 
 		log_test("\t[path_prefix]_[name] was put in data/screenshots_new")
 
+/// Helper for screenshot tests to take an image of an atom from all directions and insert it into one icon
+/datum/unit_test/proc/get_flat_icon_for_all_directions(atom/thing, no_anim = TRUE)
```

Comment:
should probably be in icons.dm or whatever that helpers file is, it's fairly generic